### ISSUE TITLE
Connecting databases between apps

### ISSUE THREAD

**Issue by roicort** [pluralsh/plural-artifacts#669]
## What content is missing

Is there a way to connect the db of one app to another app? 

For example connecting an app db to AirByte to create a ETL
or another app db to Metabase to view analytics.

---
#### Message from the maintainers:
Are you looking for the same documentation content? Give it a :thumbsup:. We factor engagement into prioritization.

**Comment by michaeljguarino** [comment]
Hey I believe both of those applications should have support for that built in, is the issue network access to the app db?

**Comment by roicort** [comment]
My apologies. Yes, how would it be in terms of networking within the Plural Cluster.

**Comment by michaeljguarino** [comment]
We default to creating the cluster within its own vpc, so if those app dbs are located in a private network within their own vpc you might need to peer the two: https://docs.plural.sh/operations/network-configuration#configuring-vpc-peering

**Comment by roicort** [comment]
Yes, what I'm trying to accomplish is connecting one app database to another app using the Postgres connection parameters. For example, if I deposit information con HASURA default database how do I get the connection parameters for to connect Postgres to other app (for example Metabase to analyze them)?

**Comment by roicort** [comment]
For example, this tutorial provides information about how to connect to a db using the console. https://docs.plural.sh/operations/managing-applications/connect-application-db But, is there a way to get the connection parameters to the app database? I mean, user, password, ip within the vpc.
